packages/image-block/manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+    "appId": ""
```

Comment:
could you already ask luca to create an app within the marketplace, so we can deploy the block for review?

Reply:
I've created an app and pushed a first release: https://product.frontify.com/marketplace/app/cldju6bkp0001a8w43x1fgzzu?mode=view